5740-2020-Robot/src/main/java/frc/robot/Robot.java

```diff
@@ -36,6 +41,9 @@ public void robotInit() {
     // autonomous chooser on the dashboard.
 
     m_robotContainer = new RobotContainer();
+    m_testPath = new TestPath(m_robotContainer.getDrivetrain());
+    m_drivetrain = m_robotContainer.getDrivetrain();
```

Comment:
reverse the order of these 2 lines and use m_drivetrain as the argument to TestPath.